conans/util/tracer.py

```diff
@@ -168,11 +127,13 @@ def log_client_rest_api_call(url, method, duration, headers):
                                      "duration": duration, "headers": headers})
 
 
+def log_conan_api_call(name, kwargs):
```

Comment:
Maybe a lot of indirections? Calling something in the tracer, that appends action, to finally call the ``trace()`` method? Maybe it is just an extra argument in the ``trace()`` method that allows to get passed "CONAN_API"? (Which might be renamed to HTTP API, maybe, to make it clearly different to the PythonAPI in 2.0?)

Reply:
Sure I was trying to keep this limited in changes. But I will bring the bulldozer.

Reply:
It is fine to keep it like this in this PR, makes sense, can be done in a later PR, I was trying to see if I was missing some intent. As you prefer, of course.